torch/testing/_core.py

```diff
@@ -275,6 +276,111 @@ def make_non_contiguous(tensor: torch.Tensor) -> torch.Tensor:
     return input.data
 
 
+def make_tensor(size, device: torch.device, dtype: torch.dtype, *, low=None, high=None,
+                requires_grad: bool = False, noncontiguous: bool = False,
```

Comment:
```suggestion
                requires_grad: bool = False, non_contiguous: bool = False,
```

Reply:
This will be a big change I guess, and I'm not sure if we want to have it in this PR. `noncontiguous` is used in different files like `test/test_linalg.py`: https://github.com/pytorch/pytorch/blob/1022443168b5fad55bbd03d087abf574c9d2e9df/test/test_linalg.py#L4242.

Do you think this change is required, and if yes - should be done in this PR?

cc: @mruberry

Reply:
I would prefer it for the public interface, yes. That being said, we can simply add this to our compatibility function in `torch.testing._internal.common_utils`:

```python
def make_tensor(size, device, dtype, *, noncontiguous=False, **kwargs):
    return _make_tensor(size, device=device, dtype=dtype, non_contiguous=noncontiguous, **kwargs)
```